tests/test_plan.md

```diff
@@ -0,0 +1,20 @@
+## Manual test scenarios : 
+
+### Scenario 1 ("check that lerta doesn't have fake alerts")
+- Get random error or warnning alert,and get the following parameters from error :(Environment(En1)-Event(Ev1)-Source(S1)).
+- Check that the Environment(Ev1) has this event(Ev1) with right source(S1).
```

Comment:
source is md5(uid of message and health-check(guid)) not correct to check against